systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -201,13 +202,17 @@ public static JsonArray expectedServiceDiscoveryInfo(String plainAuth, String tl
     public static boolean checkLogForJSONFormat(Map<String, String> pods, String containerName) {
         boolean isJSON = false;
 
+        String regex = "(\\{.*})\\{";
+        Pattern pattern = Pattern.compile(regex);
```

Comment:
Can this be compiled as a static variable instead of doing it at every run?

Reply:
Fixed, thanks for the comment.